website/src/docs/webcam.md

```diff
@@ -156,6 +157,12 @@ Set the preferred mime type for images, for example `'image/png'`. If the browse
 
 If no preferred image mime type is given, the Webcam plugin will prefer types listed in the [`allowedFileTypes` restriction](/docs/uppy/#restrictions), if any.
 
+### `mobileNativeCamera`
+
+Replaces Uppy’s custom camera UI on mobile with “Take Picture” and/or “Record Video” buttons that open native device UI for pictures / video (`Function: boolean` || `boolean`, default: `isMobile()`).
```

Comment:
Is it really "and/or"? Can you disable one or the other?

Reply:
Not currently, but since we allow that for the webcam plugin, I thought we need to mimic that functionality here as well.

Reply:
Added `modes` support, please take a look @Murderlon